rpctest/rpc_harness.go

```diff
@@ -317,6 +326,15 @@ func (h *Harness) CreateTransaction(targetOutputs []*wire.TxOut,
 	return h.wallet.CreateTransaction(targetOutputs, feeRate)
 }
 
+// FreeLockedOutputs unlocks any outputs which were previously marked as
+// unspendabe due to being selected to fund a transaction via the
+// CreateTransaction method.
+//
+// This function is safe for concurrent access.
+func (h *Harness) FreeLockedOutputs(inputs []*wire.TxIn) {
```

Comment:
Same deal here if the name of the wallet method is changed to `UnlockOutputs`.  And the docs above then could say `are locked in order to avoid potential ... MUST be unlocked via a call to UnlockOutputs ...`